frontend/app/templates/guide.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="container board">
+    <div class="row">
+        <div class="jumbotron text-center">
+            <h1>Badgeyay User-Input Guide</h1>
+        </div>
+        <div class="col-sm-12 text-center">
+            <p>Want to learn what is the "Correct Format"?</p>
+            <p>Check
+                <a href="#csv">CSV</a>,
+                <a href="#manual">Manual Data</a>,
+            </p>
+        </div>
+        <div class="col-sm-12 text-left options" id="csv">
+            <h2>1. CSV Format</h2>
+            <pre class="alert-info">The CSV must be uploaded with 5 columns of data</pre>
+            <p>Five comma (',') seperated values should be present in the CSV</p>
+            <p>See Example Image Below</p>
+            <img src="/images/guide_csv.png" class="guide_img" alt="CSV Image">
+        </div>
+        <div class="col-sm-12 text-left options" id="manual">
+            <h2>2. Manual Data Format</h2>
+            <pre class="alert-info">Format for Manual data is same as CSV's data</pre>
+            <p>Five comma (',') seperated values on each line is the correct format</p>
+            <p>See Image below</p>
+            <img src="/images/guide_manual.png" class="guide_img" alt="Manual Data Image">
```

Comment:
Same as above, not correct location to image.

Reply:
The above one is the same file.

Reply:
sorry those files were not added,Now I have added them ,Please check thank you